src/OSPSuite.Core/Domain/Container.cs

```diff
@@ -24,6 +24,11 @@ public interface IContainer : IEntity, IEnumerable<IEntity>
       /// </summary>
       IReadOnlyList<IEntity> Children { get; }
 
+      /// <summary>
+      /// Returns the path to the parent container.
+      /// It should only be set if the container has no parent in the hierarchy. Otherwise, it will be null
+      /// </summary>
+      ObjectPath ParentPath { get; set; }
```

Comment:
@Yuri05 I am not quite sure about the behaviour of this ParentPath. I am not sure I want to synchronize with the ParentContainer of the Entity. I think it should be null all the time to indicate it should be displayed as default in mobi